src/main/java/ui/PMTaskInitScreen.java

```diff
@@ -0,0 +1,60 @@
+package ui;
+
+import controller.PMTaskInitController;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class PMTaskInitScreen extends JPanel implements ActionListener {
+
+    JTextField taskName = new JTextField(50);
+    JTextField taskDescription = new JTextField(50);
+    JTextField employeeId = new JTextField(50);
+    PMTaskInitController taskInitController;
+
+    public PMTaskInitScreen(PMTaskInitController controller) {
```

Comment:
I feel like you should initialize your controller and all your use case inside the constructor

Reply:
I have initialized the controller, etc. in the newly added main method. Since the constructor takes in the controller as a parameter, it cannot be initialized inside it